mmdet/models/backbones/resnet.py

```diff
@@ -29,27 +30,40 @@ def __init__(self,
                  dilation=1,
                  downsample=None,
                  style='pytorch',
-                 with_cp=False):
+                 with_cp=False,
+                 normalize=dict(type='BN'),
+                 frozen=False):
```

Comment:
The freezing behavior may be better moved to the `_freeze_layers` method of `ResNet` to make BasicBlock/Bottleneck clean.